### DISCUSSION TITLE
RLC-842A

### DISCUSSION THREAD

**Discussion by mfzar** [AT0myks/reolink-fw-archive#32] [Firmware sharing]
Firmware provided by Reolink support 

Some of my **RLC-842A** cams were shipped with this firmware, one came with old v3.1.0.764_21121717, so they sent the newest one. Support didn't provide any additional information regarding changes though.

**version v3.1.0.1643_22122317**

[official link](https://support-d.reolink.com/attachments/token/zJUE4Mb1B0pSSvEzKfxDjA8ob/?name=IPC_523D98MP.1643_22122317.RLC-842A.IMX415.8MP.AF.REOLINK.pak)

**Comment by AT0myks** [comment]
Thank you, added in 405761277f194774bf27ea9ea8e09d73c8510f78.
